apps/server/src/modules/learnroom/service/board-copy.service.ts

```diff
@@ -9,10 +9,14 @@ import {
 	Course,
 	Lesson,
 	Task,
-	TaskCopyService,
 	User,
 } from '@shared/domain';
+import { BoardRepo } from '@shared/repo';
+import { Logger } from '@src/core/logger';
+import { getResolvedValues } from '@src/modules/files-storage/helper';
+import { sortBy } from 'lodash';
 import { LessonCopyService } from './lesson-copy.service';
+import { TaskCopyService } from './task-copy.service';
 
 export type BoardCopyParams = {
```

Comment:
is it common in this project to not have types defined in a specific place? defining types in any file could lead to quite a lot of chaos, no?

Reply:
It is common in the initiative for a few copy functions and it's only the interface of this single function.
(Btw. discussing the problem with uwe in BBB)